recipes/agent_build_tools.rb

```diff
@@ -52,8 +52,44 @@
   only_if { has_nvidia_support? }
 end
 
-# GeForce GTX 550 Ti requires old 3xx.xx series
-package 'nvidia-384' do
+if has_nvidia_support? and nvidia_devices.size != 1
+  Chef::Log.warn("There are multiple nvidia devices and I am only looking at the first!")
+end
+nvidia_device = if has_nvidia_support?
+                  nvidia_devices.first['device']
+                else
+                  nil
+                end
```

Comment:
An empty `else` clause would technically evaluate to `nil` anyway but since we're using it as a sigil value below I thought it best to be explicit.